app/assets/javascripts/components/locales/index.jsx

```diff
@@ -22,6 +22,7 @@ import zh_hk from './zh-hk';
 import zh_cn from './zh-cn';
 import bg from './bg';
 import id from './id';
+import kab from './kab';
```

Comment:
Hi @belkacem77 thank you very much for answering my e-mail :)
And welcome to kabyle language into Mastodon :)
If you need any help just ping me :)
Tanmirt ik.